lib/order.rb

```diff
@@ -0,0 +1,20 @@
+class Order
+  attr_reader :basket, :total
+
+  def initialize
+    @basket = Hash.new(0)
+  end
+
+  def add_basket(item, quantity)
+    basket[item] += quantity
+  end
+
+  def basket_summary(menu)
+    basket.map { |item, quantity| "#{item} x#{quantity} = £#{sprintf('%.2f', menu.dishes[item] * quantity)}" }.join(', ')
+  end
+
+  def total(menu)
+    @total = order.basket.map { |item, quantity| menu.dishes[item] * quantity }.inject(:+)
```

Comment:
Metrics/LineLength: Line is too long. [90/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)